transit_odp/common/templatetags/filters.py

```diff
@@ -26,6 +26,12 @@ def percentage(text, arg=-1):
     return floatformat(percentage_value, arg=arg) + "%"
 
 
+@register.filter()
+def subtract(value1, value2):
```

Comment:
Do we need to handle negative numbers?

Reply:
we need the difference between two numbers